src/app/pages/unuploaded-tests/unuploaded-tests.page.ts

```diff
@@ -67,5 +67,5 @@ export class UnuploadedTestsPage implements OnInit {
 
   getEmployeeNumberDisplayValue = (employeeNumber: string): string => employeeNumber || 'NOT_KNOWN';
 
-  getTestsText = (tests: number): string => tests > 1 ? 'tests' : 'test';
+  getTestsText = (tests: number): string => (tests === 0 || tests > 1) ? 'tests' : 'test';
```

Comment:
Is this the same as `getTestsText = (tests: number): string => tests === 1 ? 'test' : 'tests';`?